src/components/pages/frequently_asked_questions/Question.vue

```diff
@@ -1,38 +1,44 @@
 <template>
-	<div class="accordion-item" v-bind:class="[ isOpen ? 'accordion-open' : '' ]">
-		<div @click="isOpen = !isOpen"
+	<div class="accordion-item" :class="[isOpen ? 'accordion-open' : '']">
+		<div
+			@click="isOpen = !isOpen"
 			:data-content-target="isOpen ? '/page/Häufige Fragen' : ''"
 			:data-track-content="isOpen"
 			:data-content-name="isOpen ? 'Expand' : ''"
-			:data-content-piece="isOpen ? content.question : ''">
-			<div v-bind:class="[ isOpen ? 'has-text-primary' : 'accordion-heading', 'icon-inline', 'accordion-title' ] ">
+			:data-content-piece="isOpen ? content.question : ''"
+		>
+			<div :class="[isOpen ? 'has-text-primary' : 'accordion-heading', 'icon-inline', 'accordion-title']">
 				{{ content.question }}
-				<ArrowUp v-if="isOpen"/>
-				<ArrowDown v-else/>
+				<ArrowUp v-if="isOpen" />
+				<ArrowDown v-else />
 			</div>
 		</div>
 		<div v-show="isOpen" v-html="content.visibleText" class="accordion-content"></div>
 	</div>
 </template>
 
+<script setup lang="ts">
+import { QuestionModel } from '@src/view_models/faq';
+
+defineProps( {
+	content: {
+		type: Object as () => QuestionModel,
+	},
+} );
+
+let isOpen = false;
+</script>
+
 <script lang="ts">
 import { defineComponent } from 'vue';
-import { QuestionModel } from '@src/view_models/faq';
 import ArrowUp from '@src/components/shared/icons/ArrowUp.vue';
 import ArrowDown from '@src/components/shared/icons/ArrowDown.vue';
 
 export default defineComponent( {
 	name: 'question',
-	components: { ArrowDown, ArrowUp },
-	props: {
-		content: {
-			type: Object as () => QuestionModel,
-		},
-	},
-	data() {
-		return {
-			isOpen: false,
-		};
+	components: {
+		ArrowDown,
+		ArrowUp,
```

Comment:
Sorry, we already converted `Question` as part of #217 (see https://github.com/wmde/fundraising-app-frontend/pull/217/files#diff-1c1a64ca9b3308236d3de90a8e28b374fa4dd534339e4ed161302b9b79fa35a5 )